### ISSUE TITLE
Duplicate page title: 1

### ISSUE THREAD

**Issue by jwilk** [jwilk/pdf2djvu#113]
**Issue reported by @juanfra684**:

```console
$ pdf2djvu 2012.05.23.a4.pdf -o output.djvu -v
Warning: /tmp//pdf2djvu.8IFFYL: Directory not empty
Duplicate page title: 1
$ pdf2djvu 2012.05.23.a4.pdf -o output.djvu
Warning: /tmp//pdf2djvu.Vwtqzo: Directory not empty
Duplicate page title: 1
```

pdf2djvu 0.9.2 on OpenBSD-current/amd64:

* GraphicsMagick-1.3.22
* poppler-0.37.0
* djvulibre-3.5.27

I've attached the problematic pdf file. It works fine with pdf2djvu-0.8.1.

<hr>

Attachment: [2012.05.23.a4.pdf](https://github.com/jwilk/pdf2djvu/files/614971/2012.05.23.a4.pdf)

**Comment by jwilk** [comment]
> What I'm contemplating is to simply leave the page without title if there was a prior page with the same title.

This has been implemented in fbf00c2db1b729a6d1ec563b22ef302483fc8187.

**Comment by jwilk** [comment]
> What I'm contemplating is to simply leave the page without title if there was a prior page with the same title.

This has been implemented in 1b262b90854cd3d5359cd7fdf6b72642b54c8f60.

**Comment by jwilk** [comment]
Fixed in 0.9.5.
